packages/xsnap/src/avaXS.js

```diff
@@ -142,6 +141,8 @@ async function runTestScript(
       assertionStatus[msg.status] += 1;
       if (msg.status === 'not ok') {
         console.warn({ ...msg, filename, label });
+        testStatus.total += 1;
```

Comment:
This seems to mix up counting tests with counting assertions. I wonder why the logic below here doesn't suffice.

Reply:
a thought: 3472734ac